core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -1157,6 +1181,16 @@ public SELF withCreateContainerCmdModifier(Consumer<CreateContainerCmd> modifier
         return self();
     }
 
+    /**
+     * Size of /dev/shm
+     * @param bytes The number of megabybytestes to assign the shared memory. Null to not set any.
```

Comment:
```suggestion
     * @param bytes The number of megabytes to assign the shared memory. Null to not set any.
```

Doesn't `null` mean we will use the Docker default of 64MB?